components/fxa-client/src/lib.rs

```diff
@@ -84,6 +83,7 @@ pub(crate) struct StateV2 {
     #[serde(default)] // Same
     device_capabilities: HashSet<DeviceCapability>,
     session_token: Option<String>, // Hex-formatted string.
+    last_seen_profile: Option<CachedResponse<Profile>>,
```

Comment:
This change doesn't need a `StateV3` as a JSON string without this will just end up being `None`.